src/pages/subjects/Subjects.tsx

```diff
@@ -125,6 +122,13 @@ const Subjects = () => {
     />
   )
 
+  const options = useMemo(
```

Comment:
what is the benefit of using this func here instead of transform?

Reply:
It does the same thing.